src/docs/BUILD.md

```diff
@@ -0,0 +1,46 @@
+# How To build OpenVINO™ GenAI
+
+## Build for Linux systems
+
+### Software requirements 
+
+- [CMake](https://cmake.org/download/) 3.23 or higher
+- GCC 7.5 or higher
+- Python 3.8 or higher
+
+### Build Instructions
+
+1. Clone OpenVINO GenAI repository and init submodules:
+    ```sh
+    git clone https://github.com/openvinotoolkit/openvino.genai.git
+    cd openvino.genai
+    git submodule update --init --recursive
+    ```
+2. Download OpenVINO archive and install dependencies:
+    ```sh
+    mkdir ./ov/
+    curl https://storage.openvinotoolkit.org/repositories/openvino/packages/pre-release/2024.2.0rc1/linux/l_openvino_toolkit_ubuntu20_2024.2.0.dev20240524_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
+    sudo ./ov/install_dependencies/install_openvino_dependencies.sh
+    ```
+3. Build the project:
+    ```sh
+    source ./ov/setupvars.sh
+    cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
+    cmake --build ./build/ --config Release --target package -j
+    cmake --install ./build/ --config Release --prefix ov
+    ./ov/samples/cpp/build_samples.sh -i ./s\ pace
```

Comment:
`s pace` is used in tests to verify our cmake vars handle it correctly. You shouldn't encourage having spaces.

I'm also not sure whether installation is part of the build process. Samples can be run from the build folder. They are installed to `ov` in tests to verify, that it will work from the package.